core/jazz_services/config/stg-config.json

```diff
@@ -3,8 +3,8 @@
     "service_required_fields": ["service", "domain", "type", "created_by", "runtime", "status", "deployment_targets"],
     "service_update_fields": ["description", "email", "slack_channel", "tags", "repository", "status","metadata","is_public_endpoint"],
     "service_filter_params": ["service", "domain", "region", "type", "runtime", "created_by", "status"],
-    "service_optional_fields": ["description", "region", "repository", "email", "slack_channel", "tags", "status","metadata","is_public_endpoint"],
-    "service_return_fields": ["service", "domain", "type", "created_by", "runtime", "description", "region", "repository", "email", "slack_channel", "tags", "status","metadata","is_public_endpoint", "deployment_targets"],
+    "service_optional_fields": ["description", "region", "repository", "email", "slack_channel", "tags", "status","metadata","is_public_endpoint","platform"],
```

Comment:
`platform` must be a required field.

Reply:
How about backward compatibility?  ok I will change it to require.

Reply:
fixed